sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 **New features**
 
+- Copy API is now supported to copy a model from one Form Recognizer resource to another
```

Comment:
could be my translation (😞) but what about "Support to copy a model from one Form Recognizer resource to another"

Reply:
I like your wording better